src/chrome/content/rules/Eclipse.xml

```diff
@@ -1,26 +1,71 @@
 <!--
-	Nonfunctional domains:
+	Connection refused:
+		- hono.eclipse.org
+		- leshan.eclipse.org
+		- rtsc.eclipse.org
 
-		download.eclipse.org
+	SSL error:
+		- rdf4j.eclipse.org
 
+	Timed out:
+		- archive.eclipse.org
+		- dirigible.eclipse.org
+		- download.eclipse.org
+		- recommenders.eclipse.org
+		- vorto.eclipse.org
+
+	Wrong server:
+		- build.eclipse.org
 -->
-<ruleset name="Eclipse (partial)">
 
+<ruleset name="Eclipse.org">
 	<target host="eclipse.org" />
+	<target host="www.eclipse.org" />
+	<target host="accounts.eclipse.org" />
+	<target host="api.eclipse.org" />
+	<target host="archive-mirror.eclipse.org" />
+	<target host="babel.eclipse.org" />
+	<target host="blogs.eclipse.org" />
 	<target host="bugs.eclipse.org" />
+	<target host="che.eclipse.org" />
+	<target host="ci.eclipse.org" />
 	<target host="dev.eclipse.org" />
+	<target host="ditto.eclipse.org" />
+	<target host="events.eclipse.org" />
+	<target host="forums.eclipse.org" />
 	<target host="git.eclipse.org" />
-	<target host="www.eclipse.org" />
-	<target host="wiki.eclipse.org" />
+	<target host="help.eclipse.org" />
+	<target host="hudson.eclipse.org" />
+	<target host="iot.eclipse.org" />
+	<target host="kapua.eclipse.org" />
+	<target host="live.eclipse.org" />
+	<target host="lts.eclipse.org" />
 	<target host="marketplace.eclipse.org" />
-	<target host="eclipsecon.org" />
```

Comment:
Could you split this into a separate ruleset?

Reply:
Added in #15774. Thanks!

Reply:
Added, thanks!